src/lib/core/CHIPConfig.h

```diff
@@ -1463,6 +1463,18 @@
 #define CHIP_CONFIG_ENABLE_UNSECURED_TCP_LISTEN 0
 #endif // CHIP_CONFIG_ENABLE_UNSECURED_TCP_LISTEN
 
+/**
+ *  @def CHIP_CONFIG_CERT_MAX_RDN_ATTRIBUTES
+ *
+ *  @brief
+ *    The maximum number of Relative Distinguished Name (RDN) attributes
+ *    supported by the CHIP Certificate Distinguished Names (DN).
+ *
+ */
+#ifndef CHIP_CONFIG_CERT_MAX_RDN_ATTRIBUTES
+#define CHIP_CONFIG_CERT_MAX_RDN_ATTRIBUTES 3
```

Comment:
Should this be 5 ?

Reply:
updated in my latest commit, which address new Authentication Tag attributes